libs/lb/lb-rs/src/lib.rs

```diff
@@ -173,18 +173,42 @@ impl<Client: Requester, Docs: DocumentService> CoreLib<Client, Docs> {
             ])
     }
 
+    #[instrument(level = "debug", skip_all, err(Debug))]
```

Comment:
I don't want the idea of v2 to leak up to the UI programmer as something to deal with. imo the complexity and key switching should be handled inside core